awswrangler/catalog/_get.py

```diff
@@ -347,9 +347,10 @@ def tables(
         df_dict["Database"].append(tbl["DatabaseName"])
         df_dict["Table"].append(tbl["Name"])
         df_dict["Description"].append(tbl.get("Description", ""))
-        if "Columns" in tbl["StorageDescriptor"]:
-            df_dict["Columns"].append(", ".join([x["Name"] for x in tbl["StorageDescriptor"]["Columns"]]))
-        else:
+        try:
+            columns = tbl["StorageDescriptor"]["Columns"]
```

Comment:
You're doing `.get` with a default below - should it be the same here?

Reply:
No, here I want to ask for forgiveness not permission. A get would return None by default whereas I want the operation to fail and return a `KeyError`

Reply:
What about below? Should it be consistent?

Reply:
Below operation would never fail because of the if and get which is consistent with the rest of the codebase. The above is the only one using a try because the alternative would be to have imbricated ifs which I find ugly